ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15.scala

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.api.tls.TlsConfiguration
+
+package object v1_15 {
```

Comment:
What is actually a relationship between 1.15 and 1.dev currently? Why InterfaceSubscriptionsIT is not part of 1.15?

Reply:
We will add back in ahtoner PR.

Reply:
Will add it in the next PR.